de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -160,12 +160,255 @@ Congratulations. You have successfully set up your Git integration!
+
+WARNING: When editing a concept, the version number should be increased before making any changes to the concept.
+Otherwise the changes will be considered to be part of the old version.
+Not doing this will cause automatic migration to fail!
 
 === Structural Elements
 
+_Structural Elements_ decompose a system into smaller components.
+For example, in the _CEF_ concept, a system is structurally decomposed into _System_ -> _SubSystem_ -> _Equipment_.
+Structural Elements are also the containers for data information (Categories and Properties).
+To support configuration management and to bundle common data, Structural Elements can also inherit data from one another.
+The instance of a Structural Element is called a _Structural Element Instance_.
+
+A Structural Element can have the following additional attributes:
+
+description:: 
+The description of the Structural Element.
+
+IsRootStructuralElement::
+Makes it possible to create instances directly in the _Repository_.
+
+Applicable For::
+Describes which Structural Elements may have instances of this Structural Element as a child.
+The Applicable For can have the following values:
+* A list of the form _[StructuralElement1, StructuralElement2,...]_.
+* _All_. Instances can be then attached to any type of Structural Element.
+
+Inherits From::
+Describes which type of Structural Elements, instances of this Structural Element can inherit data from.
+Note that the concrete inheritance link between instances is defined during runtime.
+At concept level, only the types between which inheritance is possible are defined.
+When an instance inherits from another instance, all attached data will be passed on to the inheriting instance.
+The Inherits From can have the following values:
+* A list of the form _[StructuralElement1, StructuralElement2,...]_.
+* _All_. Instances can then inherit from any type of Structural Element.
+
+The following example describes a simple system decomposition.
+
+----
+StructuralElement ComponentType {
+	IsRootStructuralElement;
+	Applicable For [ComponentType];
+}
+
+StructuralElement Component {
+	IsRootStructuralElement;
+	Inherits From [ComponentType];
+	Applicable For [Component];
+}
+----
+
+It defines a _ComponentType_ which can be attached to the _Repository_.
+It can also be nested.
+As a second Structural Element, the example defines a _Component_, which can also be attached to the _Repository_.
+It can be nested and inherit data from a selected _ComponentType_. 
+
+=== Categories
+
+Categories are the main way of assigning data to a Structural Element.
+The instance of a Category is called a _CategoryAssignment_.
+
+A Category may contain properties - described in the next section - and
+also supports the following attributes:
+
+extends:: 
+A Category may extend another Category.
+The extending Category will then inherit all defined properties from the extended Category.
+
+description::

```

Comment:
There is two times extends in the rendered version
![image](https://user-images.githubusercontent.com/75322624/101154965-c8c5de80-3626-11eb-8da1-025a3dc25d1c.png)